Install target not working

I was tring to install this game on my machine but i got this error:
```
Making install in src
make[1]: Entering directory '/home/grillo/repos/snaskii22/src'
/bin/sh ../libtool  --tag=CC   --mode=link gcc -pthread  -g -O2   -o snaskii.bin snaskii_bin-snaskii.o snaskii_bin-utils.o -lncurses    -lncurses
libtool: link: gcc -g -O2 -o snaskii.bin snaskii_bin-snaskii.o snaskii_bin-utils.o  -lncurses -pthread
cp snaskii.sh snaskii
chmod u+x snaskii
make[2]: Entering directory '/home/grillo/repos/snaskii22/src'
 /usr/bin/mkdir -p '/usr/local/bin'
  /bin/sh ../libtool   --mode=install /usr/bin/install -c snaskii.bin '/usr/local/bin'
libtool: install: /usr/bin/install -c snaskii.bin /usr/local/bin/snaskii.bin
 /usr/bin/mkdir -p '/usr/local/bin'
 /usr/bin/install -c snaskii '/usr/local/bin'
make  install-exec-hook
make[3]: Entering directory '/home/grillo/repos/snaskii22/src'
cd //usr/local/bin && mv ttsnake.bin ttsnake
mv: cannot stat 'ttsnake.bin': No such file or directory
make[3]: *** [Makefile:766: install-exec-hook] Error 1
make[3]: Leaving directory '/home/grillo/repos/snaskii22/src'
make[2]: *** [Makefile:693: install-exec-am] Error 2
make[2]: Leaving directory '/home/grillo/repos/snaskii22/src'
make[1]: *** [Makefile:637: install-am] Error 2
make[1]: Leaving directory '/home/grillo/repos/snaskii22/src'
make: *** [Makefile:412: install-recursive] Error 1
```
It is trying to install a program named `ttsnake` instead of `snakii`.

--- a/nix/default.nix
+++ b/nix/default.nix
@@ -21,15 +21,4 @@ stdenv.mkDerivation rec {
     autoreconf --install
   '';
 
-  installPhase = ''
-    mkdir -p $out/bin
-    chmod -R u+x $out
-
-    cp src/snaskii $out/bin
-    cp src/snaskii.bin $out/bin
-    cp -r scenes $out
-    chmod -R u+x $out/bin
-
-  '';
-
 }

--- a/src/Makefile.am
+++ b/src/Makefile.am
@@ -50,17 +50,17 @@ snaskii : snaskii.sh
 # This install snaskii.bin and rename it snaskii.
 
 install-exec-hook: 
-	cd $(DESTDIR)/$(bindir) && mv ttsnake.bin ttsnake
+	cd $(DESTDIR)/$(bindir) && mv $(bin_PROGRAMS) $(bin_SCRIPTS)
 
 # This uninstall the binary
 
 uninstall-hook:
-	rm -f $(DESTDIR)/$(bindir)/snaskii
+	rm -f $(DESTDIR)/$(bindir)/$(bin_SCRIPTS)
 
 # This removes the wrapper script.
 
 clean-local:
-	rm -f snaskii
+	rm -f $(bin_SCRIPTS)
 
 
 

